database/schemas/users.model.js

```diff
@@ -23,6 +23,10 @@ module.exports = (sequalize, type) => {
       updatedAt: {
         type: type.DATE,
       },
+      token: {
+        type: type.TEXT,
```

Comment:
text?

Reply:
Text -  I don't know how long will be the Token

Reply:
I think it can't be more than 255 symbols, so you can use varchar(255)